src/game/file_io.c

```diff
@@ -633,14 +635,18 @@ int game_file_io_read_saved_game(const char *filename, int offset)
     log_info("Loading saved game", filename, 0);
     FILE *fp = file_open(dir_get_case_corrected_file(filename), "rb");
     if (!fp) {
+        log_error("Unable to load game", 0, 0);
```

Comment:
Add filename as 2nd param so it is included in the log message.

Reply:
I didn't do that because the log will already be:

```
Loading save game: <save game>
ERROR: Unable to load game
```

Writing the file name twice feels redundant...